### PR TITLE
implement Stark cards from Someone Always Tells chapter pack

### PR DESCRIPTION
**Pull request by zacchiro**
Not to be merged yet, as we don't have neither the corresponding JSON data yet,
nor the proper english card names. For review purpose, I've kept at the
beginning of each card implementation a URL pointing to spoiler image; it
should be removed before merging too.

### CODE REVIEW

File: server/game/cards/08.6-SAT/BoisSacre.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+// https://i.redditmedia.com/2ZjgGkqjmt3xilgW4EyicxqikQOoh4WP6J-mq5cE150.jpg?w=495&s=1f409ec2df0388fcf2e19e11f90e8561
+
+const DrawCard = require('../../drawcard.js');
+
+class BoisSacre extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                afterChallenge: ({challenge}) =>
```
Comment (DukeTax) [inline server/game/cards/08.6-SAT/BoisSacre.js]: There's a bunch of these in the code, but we try not to deconstruct event objects anymore, so ```event``` and ```event.challenge.winner``` instead.

File: server/game/cards/08.6-SAT/BoisSacre.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+                    challenge.winner === this.controller && this.moreWinterThanSummerPlotsRevealed()
+            },
+            cost: ability.costs.kneelSelf(),
+            handler: () => {
+                let numCards = this.controller.activePlot.getClaim();
+                this.controller.drawCardsToHand(numCards);
+                this.game.addMessage('{0} uses {1} to draw {2} cards', this.controller, this, numCards);
```
Comment (DukeTax) [inline server/game/cards/08.6-SAT/BoisSacre.js]: uses --> kneels to keep it in line with other such costs.

File: server/game/cards/08.6-SAT/OstDuConflans.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            cost: ability.costs.discardGold(),
+            handler: context => {
+                this.game.addPower(context.event.card, 1);
+                this.game.addMessage('{0} discards a gold from {1} to have {2} gain 1 power',
+                    this.controller, this, context.event.card);
+            }
+        });
+    }
+
+    getNumberOfAttackingFreys() {
```
Comment (DukeTax) [inline server/game/cards/08.6-SAT/OstDuConflans.js]: Unused method.

**Comment by zacchiro** [comment]
This should be now good to go.
Thanks to my Paris meta-mates Utilisateur and Beren Miriel for their help in tracking down the (most likely) English card names.

**Pull request merged**